server.go

```diff
@@ -74,12 +85,11 @@ type Server struct {
 }
 
 // NewFromConfig creates a new veneur server from a configuration specification.
-func NewFromConfig(conf Config) (ret Server, err error) {
+func NewFromConfig(conf Config, transport http.RoundTripper) (ret Server, err error) {
```

Comment:
Yeah, that's a good point. My problems are only because `NewFromConfig` is starting things. Will think on it too!